qgepplugin/gui/qgepdatamodeldialog.py

```diff
@@ -0,0 +1,773 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------
+#
+# Profile
+# Copyright (C) 2012  Matthias Kuhn
+# -----------------------------------------------------------
+#
+# licensed under the terms of GNU GPL 2
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, print to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# ---------------------------------------------------------------------
+
+import os
+import sys
+import configparser
+import functools
+import zipfile
+import tempfile
+import pkg_resources
+import subprocess
+import psycopg2
+
+from qgis.PyQt.QtWidgets import QDialog, QMessageBox, QProgressDialog, QApplication, QPushButton
+from qgis.PyQt.QtCore import QUrl, QFile, QIODevice
+from qgis.PyQt.QtNetwork import QNetworkRequest, QNetworkReply
+
+from qgis.core import QgsMessageLog, QgsNetworkAccessManager, Qgis, QgsProject
+
+from ..utils import get_ui_class
+
+# TODO : get latest dynamically ?
+AVAILABLE_RELEASES = {
+    # 'master': 'https://github.com/QGEP/datamodel/archive/master.zip',  # TODO : if we expose this here, we should put a big red warning and not take it default
+    '1.5.3': 'https://github.com/QGEP/datamodel/archive/1.5.3.zip',
```

Comment:
```suggestion
    '1.5.4': 'https://github.com/QGEP/datamodel/archive/1.5.4.zip',
```

I guess that proves the point of line 44

Reply:
Added a note about that :
```
# Currently, the latest release is hard-coded in the plugin, meaning we need
# to publish a plugin update for each datamodel update.
# In the future, once plugin/datamodel versionning scheme clearly reflects
# compatibility, we could retrieve this dynamically, so datamodel bugfix
# releases don't require a plugin upgrade.
```

What do you think ?